client/state/form/packages/reducer.js

```diff
@@ -10,6 +10,10 @@ import {
 import omitBy from 'lodash/omitBy';
 import isNull from 'lodash/isNull';
 
+const isNullOrEmpty = ( value ) => {
```

Comment:
Good point. I was initially thrown off by it returning true for values like `12`, but it does work fine for strings.